.travis.yml

```diff
@@ -68,13 +68,13 @@ script:
     fi
   - tests/Fixtures/app/console cache:clear
   - if [[ $APP_ENV = 'postgres' ]]; then
-      vendor/bin/behat --suite=postgres --format=progress;
+      vendor/bin/behat --suite=postgres --format=progress --no-interaction;
```

Comment:
why?

Reply:
Undefined step `Then this is not supported`, and because `behat` CLI can't tell correctly when it's in a non-interactive context. :see_no_evil: